add_me_to_your_PATH/julia_pod

```diff
@@ -202,6 +198,10 @@ kubectl attach "pod/$podname" -c driver -it -n "$KUBERNETES_NAMESPACE" && echo "
     log "Detached from pod, killing folder syncs and/or port forwarding!"
     echo ""
 
+    if kubectl get secret $SECRET_NAME &>/dev/null; then
+        kubectl delete secret $SECRET_NAME
+    fi
```

Comment:
This just moves the problem to occur when `julia_pod`s being stopped and started at the same time